clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/utils/AwsLookupUtil.groovy

```diff
@@ -27,23 +26,25 @@ import com.netflix.spinnaker.clouddriver.aws.provider.view.AmazonVpcProvider
 import com.netflix.spinnaker.clouddriver.aws.security.AmazonCredentials
 import com.netflix.spinnaker.clouddriver.aws.services.RegionScopedProviderFactory
 import com.netflix.spinnaker.clouddriver.security.AccountCredentialsProvider
+import com.netflix.spinnaker.clouddriver.titus.caching.Keys
 import com.netflix.spinnaker.clouddriver.titus.client.model.Job
 import com.netflix.spinnaker.clouddriver.titus.credentials.NetflixTitusCredentials
 import com.netflix.spinnaker.clouddriver.titus.model.TitusInstance
-import com.netflix.spinnaker.clouddriver.titus.model.TitusSecurityGroup
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Component
 
 import javax.annotation.PostConstruct
 
-import static com.netflix.spinnaker.clouddriver.aws.cache.Keys.Namespace.SECURITY_GROUPS
 import static com.netflix.spinnaker.clouddriver.core.provider.agent.Namespace.HEALTH
+import static com.netflix.spinnaker.clouddriver.titus.caching.Keys.CachingSchema.V1
```

Comment:
Would prefer for the caching schema logic to be in its own utility class since this is going to be very likely very ephemeral. A titus region feature flag feels more appropiate

Reply:
Moving caching schema logic to separate class. I chose to flag at account level and not region because of on how we query for things: account is often specified, whereas region is not. Seemed less error-prone to add an account level flag rather than adding a check to make sure that if the split flag is enabled in one region it's enabled everywhere. 

I'll remove the flag/config for it when we switch over completely.